.env

```diff
@@ -2,4 +2,4 @@ OPENSLIDES_DATASTORE_READER_PORT=8001
 OPENSLIDES_DATASTORE_WRITER_PORT=8000
 DATASTORE_DATABASE_PORT=5432
 MESSAGE_BUS_PORT=6379
-OPENSLIDES_DATASTORE_SERVICE_COMMIT_HASH=fuse-dockerfiles
+OPENSLIDES_DATASTORE_SERVICE_COMMIT_HASH=master
```

Comment:
Maybe rename it. I tried it and though, a commit_hash has to be used. But a branch name is required. If you use a commit_hash, then the build fails.